### PR TITLE
Update `name` and `version` fields in block workspaces

### PR DESCRIPTION
**Pull request by kachkaev**
## 🌟 What is the purpose of this PR?

I was going to align our yarn workspaces with blocks as something like `@hashintel/block-*`, but discovered that some blocks (like `calculation`) did not belong to https://blockprotocol.org/@hash. I went checking the `@blocks/*` namespace on NPM and concluded that it was used by some private customer.

Because it is impossible for anyone to publish `@blocks/*` packages on npmjs.com, I’ve decided to move all block workspaces to this namespace. Using `@blocks/*` makes it simple to match workspace with folder names, which improves DX. If we ever need to give up `@blocks/*` namespace for yarn workspaces, we can use `@bp-blocks/*` or alike.

In addition to renaming workspaces, I’ve also changed few block versions from `0.0.0` with `0.1.0`. Seeing `0.0.0` in the hub is a bit confusing.

I understand that we might want to publish some code from some blocks as npm packages. If this happens, we’ll probably want to extract that code into a package and then implement a block around the package. We already do this for the calculation block.

The result is tested in https://github.com/blockprotocol/blockprotocol/pull/733.

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1201095311341924/1203294246816660/f) _(internal)_

## 📜 Does this require a change to the docs?

- No

## 🐾 Next steps

- #1368
- Rename and move packages in other workspaces

## ❓ How to test this?

1.  Check CI
1.  Check https://github.com/blockprotocol/blockprotocol/pull/733

### CODE REVIEW

File: blocks/stopwatch/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "stopwatch",
```
Comment (kachkaev) [inline blocks/stopwatch/package.json:2]: Not present on https://blockprotocol.org/hub 🤔

File: blocks/toggle-item/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "toggle-item-block",
```
Comment (kachkaev) [inline blocks/toggle-item/package.json:2]: Not present on https://blockprotocol.org/hub 🤔

**Review by CiaranMn — APPROVED** [review]

**Comment by nonparibus** [comment]
I understand the motivation/rationale given in the description, butI don’t think we should do this. We should register `@hashblocks` instead, or equivalent, or we really think a `@hash/block-*` prefix is so bad (which I am skeptical of, provided we use it consistently to refer to all blocks in our `/blocks` folder).

**Review by nonparibus — DISMISSED** [review]
I don't think this is the right solution. See above.

**Pull request merged**